README.md

```diff
@@ -77,7 +77,7 @@ ACPMedia.extensionVersion().then(version => console.log("AdobeExperienceSDK: ACP
 
 ##### Registering the extension with ACPCore:
 
-> Note: It is recommended to initialize the SDK via native code inside your AppDelegate and MainApplication in iOS and Android respectively. However, you can still initialize the SDK in Javascript. For more information see how to initialize [Core](https://github.com/adobe/react-native-acpcore#initializing-the-sdk).
+> Note: It is recommended to initialize the SDK via native code inside your AppDelegate and MainApplication in iOS and Android respectively. However, you can still initialize the SDK in Javascript.
```

Comment:
Should remove `However, you can still initialize the SDK in Javascript.`

Reply:
Thanks for catching that.